source/robot_model/include/robot_model/Model.hpp

```diff
@@ -98,6 +110,34 @@ class Model {
    */
   unsigned int get_frame_id(const std::string& frame);
 
+  /**
+   * @brief Function that extracts package names from URDF string
+   * @param urdf string containing the URDF description of the robot
+   * @return set of package names
+  */
+  std::set<std::string> extract_package_name_from_urdf();
+
+  /**
+   * @brief Replaces the package name by the full path in the urdf file
+   * @param urdf string containing the URDF description of the robot
+   * @param package_name the name of the package to replace
+   * @param full_path the full path to replace the package name
+  */
+  void replace_package_by_full_path(const std::string& package_name, const std::string& full_path);
+
+  /**
+    * @brief Resolves the package paths in the URDF string and returns the resolved URDF string
+    * @param urdf string containing the URDF description of the robot
+  **/
+  void resolve_package_paths();
+
+  /**
+    @brief Reads the URDF file from the path and returns the URDF string
```

Comment:
nitpick: use the imperative case for docstrings. Because this documentation is [generated and publicly hosted](https://aica-technology.github.io/control-libraries/versions/main/), it's important to be clear and consistent. 

```suggestion
   * @brief Extract package names from URDF string
   * @param urdf string containing the URDF description of the robot
   * @return set of package names
  */
  std::set<std::string> extract_package_name_from_urdf();

  /**
   * @brief Replace the package name by the full path in the urdf file
   * @param urdf string containing the URDF description of the robot
   * @param package_name the name of the package to replace
   * @param full_path the full path to replace the package name
  */
  void replace_package_by_full_path(const std::string& package_name, const std::string& full_path);

  /**
    * @brief Resolve the package paths in the URDF string and return the resolved URDF string
    * @param urdf string containing the URDF description of the robot
  **/
  void resolve_package_paths();

  /**
    @brief Read the URDF file from the path and return the URDF string
```